app/js/components/tabs/tabcomponents/patientComponent.jsx

```diff
@@ -180,10 +207,10 @@ class PatientComponent extends Component {
                         {
                             this.state.toDisplay.map(patient => {
                                 return (
-                                    <tr key={patient.uuid}>
-                                        <td>{patient.person.display}</td>
-                                        <td>{patient.person.age}</td>
-                                        <td>{patient.person.gender}</td>
+                                    <tr key={shortId.generate()}>
+                                        <td>{`${patient.Firstname} ${patient.Lastname}`}</td>
```

Comment:
Our convention has always been to start with small letters for variable names. Any reason for using Firstname, instead of firstname, etc?